README.md

```diff
@@ -26,10 +26,48 @@ This image can be built and operated behind a corporate proxy where the base os
 
 While building locally using the Makefile, you may set the environment variable CUSTOM_CA to a file or directory in order to import them.
 `CUSTOM_CA=/usr/local/share/ca-certificates make`
+## Run locally
+Image could be run locally using vagrant box [fredrikhgrelland/hashistack](https://github.com/fredrikhgrelland/vagrant-hashistack). 
 
-## Examples
-TBD
+Prerequisite is to have [vagrant](https://www.vagrantup.com/) Prerequisite is to have [vagrant](https://www.vagrantup.com/) and [virtualbox](https://www.virtualbox.org/) 
```

Comment:
Should we add a prereq target in makefile?

Reply:
I would like to avoid it, due that is the context of `vagrant-hashistack`. Maybe link would be enough? https://github.com/fredrikhgrelland/vagrant-hashistack#build--test

Reply:
I imagine a `make prereq` would be needed in `data-mesh, and even thoughtThat might not be the same for `docker-hive`, I would like to extend that to `docker-hive` for consistency.

Reply:
Yes. Maybe that should be a complete separate section of fredrikhgrelland/vagrant-hashistack devoted to installation ( and subsequent update? )

Reply:
ok, I will fix it.